src/components/ComputeModal.js

```diff
@@ -1001,8 +1001,7 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
           id => div({ style: { gridColumnEnd: 'span 3' } }, [
             label({ htmlFor: id, style: computeStyles.label }, ['Location']),
             h(InfoBox, { style: { marginLeft: '0.5rem' } }, [
-              'By default, workflow and cloud environments will run in the same region as the workspace bucket. ',
-              p(['Currently, only a limited amount of locations are available but more will become available over time.'])
+              'Cloud environments run in the same region as the workspace bucket and cannot be changed at this time.',
```

Comment:
Suggestion: where you are changing things that will need to be updated with subsequent commits, it would be nice to include a comment with the JIRA ticket for the future work, so that it easy to search for the places that need to be updated. If there are not JIRA tickets for each of the planned "next steps", it would be good to create them.